scripts/accounts/gcloud/verify-principal-roles-and-permissions.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+while getopts g:s:p:r:f:e:i: flag
+do
+    case "${flag}" in
+        g) google_account=${OPTARG};;
+	s) service_account=${OPTARG};;
+        p) project_id=${OPTARG};;
+        r) roles=${OPTARG};;
+        f) roles_file=${OPTARG};;
+        e) permissions=${OPTARG};;
+	i) permissions_file=${OPTARG};;
+    esac
+done
+
+if [ "$1" == "-h" ];
+then
+    echo "Checks if a Principal has the specified roles and permissions in a given project."
+    echo ""
+    echo "Arguments:"
+    echo "  -g     [Required] Google Account for the end user. (mutual exclusive with -s)"
+    echo "  -s     [Required] Service Account name. (mutual exclusive with -g)"
+    echo "  -p     [Required] Project ID where the roles and permissions will be checked"
+    echo "  -r     Roles to be checked for the principal in the project, splitted by comma."
+    echo "  -f     Path to a file containing the roles to be checked for the principal in the project."
+    echo "  -e     Permissions to be checked for the principal in the project, splitted by comma."
+    echo "  -i     Path to a file containing the permissions to be checked for the principal in the project."
```

Comment:
```suggestion
    echo "  -g     [Required] Google Account of an end user. Mutually exclusive with -s."
    echo "  -s     [Required] Service Account name. Mutually exclusive with -g."
    echo "  -p     [Required] Short project name (ID) where the roles and permissions will be checked."
    echo "  -r                Roles to be checked, splitted by comma."
    echo "  -f                Path to a file containing the roles to be checked."
    echo "  -e                Permissions to be checked, splitted by comma."
    echo "  -i                Path to a file containing the permissions to be checked."
```

Update documentation accordingly.